src/adapter/src/coord.rs

```diff
@@ -2347,6 +2350,7 @@ pub fn serve(
         let span = tracing::Span::current();
         let coord_now = now.clone();
         let advance_timelines_interval = tokio::time::interval(catalog.config().timestamp_interval);
+        let catalog_transient_revision = Arc::new(AtomicU64::new(catalog.transient_revision()));
```

Comment:
Hrm I'm not a fan of having both this and the revision on the `Catalog` struct. Can we brainstorm some alternatives?